azext_edge/tests/edge/checks/conftest.py

```diff
@@ -241,3 +243,25 @@ def _handle_list_deployments(*args, **kwargs):
     mocked_client.AppsV1Api().list_deployment_for_all_namespaces.side_effect = _handle_list_deployments
 
     yield mocked_client
+
+
+@pytest.fixture
+def mocked_validate_ref(mocker):
+    def _handle_validate_ref(*args, **kwargs):
+        ref_text = ""
+        if kwargs["ref_type"] == ValidationResourceType.secret:
+            ref_text = (
+                f"[green]Valid[/green] {ValidationResourceType.secret.value} reference {{[green]mock_secret[/green]}}."
+            )
+        elif kwargs["ref_type"] == ValidationResourceType.configmap:
+            ref_text = f"[green]Valid[/green] {ValidationResourceType.configmap.value}\
+                reference {{[green]mock_configmap[/green]}}."
```

Comment:
end the string before you split it, otherwise the indentation is grabbed as extra spaces:
`[green]Valid[/green] configmap                reference {[green]mock_configmap[/green]}`